### PR TITLE
links: bump to 2.20.2

### PR DESCRIPTION
**Pull request by kenmays**
NOTE:  links 2.20.2 - Mouse didn't work in GUI mode on x86 platform. Fix later.
Mouse works in GUI mode for links 2.19 x86.
Mouse works in GUI mode for links 2.20.2 x86_64.

### CODE REVIEW

**Comment by Begasus** [comment]
Why close the previous PR to open a new one for the same? (didn't even had time to check on x86_64)

**Comment by kenmays** [comment]
@Begasus - Let us save the x86 mouse debugging issue for another PR. The x86_64 build worked fine for me.

File: www-client/links/links-2.20.2.recipe [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+COPYRIGHT="1999-2019 Mikulas Patocka
+	2000-2011 Petr Kulhavy, Karel Kulhavy, Martin Pergel"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.bz2"
+CHECKSUM_SHA256="4b4f07d0e6261118d1365a5a5bfa31e1eafdbd280cfae6f0e9eedfea51a2f424"
+#PATCHES="links-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 !x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86"
```
[the remark this answers is not in the document]
Comment (kenmays) [reply www-client/links/links-2.20.2.recipe]: Fixed.

**Comment by kenmays** [comment]
@Begasus - Go to pull this one. Need to fix the issue if we are going to support the x86 build.

**Pull request closed without merging**